external.bzl

```diff
@@ -612,6 +695,389 @@ def _go_dependencies():
             "https://github.com/google/brotli/archive/v1.0.7.tar.gz",
         ],
     )
+    go_repository(
+        name = "co_honnef_go_tools",
+        importpath = "honnef.co/go/tools",
+        sum = "h1:3JgtbtFHMiCmsznwGVTUWbgGov+pVqnlf1dEJTNAXeM=",
+        version = "v0.0.1-2019.2.3",
+    )
+    go_repository(
+        name = "com_github_bazelbuild_rules_go",
```

Comment:
Why does our code need a copy of `io_bazel_rules_go`?  Ditto for the rest of these additional indirect deps.  Can we prune them since we probably don't need them all for a `//...` build?

Reply:
I wrote a dummy rule which will cause gazelle to ignore the go.mod dependency.

Reply:
Note that the indirect dependencies are still present, presumably because they are used by the build files gazelle will generate for those (and thus need to have build files themselves).